builtin/rebase--helper.c

```diff
@@ -98,5 +101,7 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		return !!edit_todo_list(flags);
 	if (command == SETUP_REFLOG && argc > 1)
 		return !!setup_reflog_action(&opts, argv[1], verbose);
+	if (command == CHECKOUT_ONTO && argc > 3)
+		return !!checkout_onto(&opts, argv[1], argv[2], argv[3], verbose);
```

Comment:
The commit message still talks about 'two new flags: “onto-name” and “onto”'. It looks like it needs an update.